apps/perf-test-react-components/src/scenarios/MakeStyles.tsx

```diff
@@ -1,6 +1,7 @@
 import { mergeClasses, makeStyles, createDOMRenderer } from '@griffel/core';
 import * as React from 'react';
 
+// eslint-disable-next-line no-restricted-globals
```

Comment:
This one is probably not needed either since the rule is turned off at the package scope.

```suggestion
```